composer.json

```diff
@@ -301,5 +301,7 @@
         "branch-alias": {
             "dev-master": "0.8-dev"
         }
-    }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true
```

Comment:
I've tagged the Symplify https://github.com/symplify/symplify/releases/tag/8.2.24 with both ECS commands you made, so you can use it

Reply:
updated.